source/agora/common/crypto/Schnorr.d

```diff
@@ -142,7 +142,14 @@ private struct Message (T)
     public T     message;
 }
 
-
+///
+unittest
```

Comment:
Is this testing anything new? It probably shouldn't be wrapped in `///` unless it should be displayed in our generated docs.

Reply:
Yes so I updated the test comment.